lib/dry/view/renderer.rb

```diff
@@ -39,21 +39,23 @@ def render(path, scope, &block)
       def chdir(dirname)
         new_paths = paths.map { |path| path.chdir(dirname) }
 
-        self.class.new(new_paths, engine: engine, format: format)
+        self.class.new(new_paths, engines: engines, format: format)
       end
 
       def lookup(name)
-        template_name = "#{name}.#{format}.#{engine}"
+        engines.map do |engine|
+          template_name = "#{name}.#{format}.#{engine}"
 
-        paths.inject(false) { |result, path|
-          result || path.lookup(template_name)
-        }
+          paths.inject(false) { |result, path|
+            result || path.lookup(template_name)
+          }
+        end.reject{|f| f == false}.first
```

Comment:
Could we make it so this outer loop "short-circuit" subsequent lookups if we've already found a template? This is what I was doing with the inject and `result || lookup` stuff in that inner loop...